bin/mail/2017-01-20-outage-followup/generate-list

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env ruby
+require_relative '../mailing-common/mailing-list-utils'
+
+# Send followup email to everyone who received the origin outage email.
+
+PEGASUS_REPORTING_DB = sequel_connect CDO.pegasus_reporting_db_reader, CDO.pegasus_reporting_db_reader
+
+outage_message_id = DB[:poste_messages].where(name: '2017-01-20-outage').first[:id]
+query = "
+  SELECT contacts.name, contacts.email
+  FROM poste_deliveries
+  INNER JOIN contacts ON contacts.id = poste_deliveries.contact_id
+  WHERE message_id = #{outage_message_id}
+"
```

Comment:
Mostly for my own benefit, I'll note that `contacts.email` isn't blanked because these were all originally teachers. If some of these were (e.g.) password reset emails, the `contacts.email` would be blank after sending.